lib/sensu-handler.rb

```diff
@@ -24,14 +24,38 @@ def handle
       puts 'ignoring event -- no handler defined'
     end
 
-    # Filters exit the proccess if the event should not be handled.
-    # Implementation of the default filters is below.
 
+    # Filters exit the proccess if the event should not be handled.
+    #
+    # Filtering events is deprecated and will be removed in a future release.
+    #
     def filter
-      filter_disabled
-      filter_repeated
-      filter_silenced
-      filter_dependencies
+      if deprecated_filtering_enabled?
+        puts 'warning: event filtering in sensu-plugin is deprecated, see http://bit.ly/sensu-plugin'
+        filter_disabled
+        filter_silenced
+        filter_dependencies
```

Comment:
If I turn off depricated filtering I thought `filter_dependencies`, `filter_disabled`, and `filter_silenced` were going to stay (although filter_disabled can easily be handled by sensu itself).